backend/Backend/Startup.cs

```diff
@@ -39,7 +39,11 @@ public void ConfigureServices(IServiceCollection services)
             {
                 options.AddPolicy(name: MyAllowSpecificOrigins,
                                   builder => {
-                                      builder.WithOrigins("http://localhost:3000").AllowAnyHeader();
+                                      builder.WithOrigins(
+                                          "http://localhost:3000",
+                                          "http://127.0.0.1:3000",
+                                          "https://localhost:3000",
+                                          "https://127.0.0.1:3000").AllowAnyHeader();
 
```

Comment:
Which URL is currently in use to access the website? or We can use any of these URLs to access the website?